doc/feature-toggles.md

```diff
@@ -0,0 +1,38 @@
+## Feature Toggles
+
+Feature toggles are used the application to turn on, or off certain features of the backend.
+
+There are two kinds of feature toggles used currently.
```

Comment:
This section is confusing. Let's just document each flag specifically, including whether it's a boolean or a regex, and what the default is. Redundant documentation is easier to read.

Reply:
👍